### PR TITLE
modified and changes are done in hooksDocumentation

### PR DESCRIPTION
**Pull request by Abhishekchauhan089**
hey @aromalanil . change are made in documentation of hooks. Please check it out

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -77,6 +77,17 @@ const LocalValue = () =>{
   );
 }
 ```
+### Parameters
+1. `inputFuction` (_any_) : Input Function of the state.
+2. `delay` (_String_) : delay for the debounce.
```
Comment (aromalanil) [inline docs/README.md:82]: The delay has a type of `number`

File: docs/README.md [hunk 2]
```diff
@@ -77,6 +77,17 @@ const LocalValue = () =>{
...
+
+### Return value
+
+`[Callback, setCallback]`
```
Comment (aromalanil) [inline docs/README.md:86]: There is some misunderstanding here. What this function returns is another function. Please refer the code of  useDebounce hook

File: docs/README.md [hunk 3]
```diff
@@ -103,6 +114,18 @@ const App = () =>{
   );
 }
 ```
+### Parameters
+1. `className` (_any_) : Initial value of the state.
+
```
Comment (aromalanil) [inline docs/README.md:119]: The classname should have a type of string and. It is the class to be added to the body when it is dark mode

**Comment by aromalanil** [comment]
@Abhishekchauhan089 I have pointed out some of the corrections to make. Also almost all of the return values are incorrect. Please refer the source code of each hooks for details about their parameters and return value.

File: docs/README.md (hunk 2 above)
Comment (Abhishekchauhan089) [reply docs/README.md:86]: can you please explain me this part again

Comment (aromalanil) [reply docs/README.md:86]: Refer [this](https://github.com/aromalanil/useCustomHooks/blob/master/hooks/useDebounce.js#L27)

This hook returns a modified function, which is forced to wait a certain amount of time before running again.

**Comment by aromalanil** [comment]
@Abhishekchauhan089 Are you making the changes? Or should I do it myself?

**Comment by Abhishekchauhan089** [comment]
@aromalanil yeah. i am working

**Comment by aromalanil** [comment]
@Abhishekchauhan089 This PR will be closed if the changes are not made within 1 day.

**Comment by aromalanil** [comment]
Closing this PR due to lack of response.

**Pull request closed without merging**